frontend/Utils/geolocation.js

```diff
@@ -0,0 +1,77 @@
+import Geocode from 'react-geocode';
+
+import {
+  MAP_KEY,
+  MAP_DEFAULT_CENTER
+} from '../../config/credentials';
+
+import _ from 'lodash';
+
+// geocode configs
+Geocode.setApiKey(MAP_KEY);
+Geocode.setLanguage('en');
+
+// Credit to https://www.w3.org/2003/01/geo/test/ustowns/latlong.htm for this data
+const defaultCentersMap = {
+  'SanFrancisco': {
+    lat: 37.7749,
+    lng: -122.4194
+  },
+  'NewYorkCity': {
+    lat: 40.77,
+    lng: -73.98
+  },
+  'LosAngeles': {
+    lat: 33.93,
+    lng: -118.40
+  }
+};
+
+
+export const getDefaultCenter = () => {
+  if (defaultCentersMap[MAP_DEFAULT_CENTER]) {
+    return defaultCentersMap[MAP_DEFAULT_CENTER];
+  } else {
+    return defaultCentersMap['SanFrancisco'];
+  }
+};
+
+/**
+ * getBrowserLocation - get's the user's location from the browser navigator API
```

Comment:
will update this comment

Reply:
Ahh so you figure there's no need for ever getting the user's browser location?

Reply:
Oddly, after doing some more testing it seems like there might be a race condition somewhere with setting the browser location. I'd need to investigate it a bit further, but removing the getBrowserLocation code altogether would probably just fix it for every case. That being said, probably wouldn't get it done until tomorrow

Reply:
JK fixed the race condition!